### DISCUSSION TITLE
Design Discussion

### DISCUSSION THREAD

**Discussion by zerebos** [rauenzi/trilium.rocks#1] [Ideas]
Which of these two designs looks better for the site?

1. The current one found on https://trilium.rocks/ which is heavily inspired by another documentation website I made (https://docs.betterdiscord.app)
2. This one I showed off in the Trilium repo (video below) inspired by the default Trilium dark theme.

https://github.com/zadam/trilium/assets/6865942/d5f6d08b-ebd3-48ed-895a-4780865cda3b

For each of these which do you like better?
- Color scheme
- Layout
- Navigation
- Compact
- UX

And of course any other thoughts you may have.

Pinging you @Nriver since you starred the repository and I haven't even made an announcement about the site yet 😄 Probably only you, me, and whoever joined the Discord server have even seen the site.

**Comment by Nriver** [comment]
Color scheme 2
Layout 1
Navigation 2
Compact 1
UX 1

Wow, you made the Table of Contents work in the shared page!

**Reply by zerebos** [reply]
Hmmm that's an interesting combination of preferences... Not sure exactly how I can mix them all together!

> Wow, you made the Table of Contents work in the shared page!

Yep, with a custom implementation! But I think it works well. You can test it live on https://trilium.rocks/ of course.

**Comment by sottey** [comment]
Love that you did this! I had planned on doing this and never got around to it. You did better than I ever could have.

I hope it is ok, I own the trilium.us domain and I have set it to simply redirect to trilium.rocks.

Almost as cool is you used a .rocks domain! I used to be the Technical Evangelist for the domain registry that owned .rocks!

Thanks for doing this

**Reply by zerebos** [reply]
> You did better than I ever could have.

Thanks man, much appreciate 😄 I really want to grow the community (and the documentation) but it's hard to take that first step. Hopefully we move off Gitter 😅 

> I hope it is ok [...]

Absolutely okay!

> I used to be the Technical Evangelist for the domain registry that owned .rocks!

That's awesome! I'm glad I chose this domain, I was hesitant tbh

> Thanks for doing this

Definitely! I just hope it keeps growing.

BTW: Any thoughts on the two designs?

**Comment by sottey** [comment]
Sorry! I forgot to add that...

For every criteria, I prefer the one in the video. It feels more "web-ish". The current site feels like a site that is generated by a notes app :-)

**Comment by zerebos** [comment]
> The current site feels like a site that is generated by a notes app

https://docs.betterdiscord.app/ in shambles 😭 


But I actually think I agree in preferring it overall. Even if it doesn't become the new default share, it's what I'll use for the site

**Reply by zerebos** [reply]
@Nriver @sottey I have updated https://trilium.rocks/ to use the new layout using custom templates. I'll be making more changes using custom templates to make the site even nicer and need less client-side js.

**Reply by sottey** [reply]
It looks so gooood!

**Reply by zerebos** [reply]
Thanks 😄 I'm particularly proud of getting search working, but I'm really liking the look of the site. It even looks good on mobile!

Only thing left is the hard part... filling it with content

**Reply by Nriver** [reply]
Great! You have updated for mobile, it looks nice now.

**Reply by zerebos** [reply]
I'm glad you like it 😄 always open to suggestions for improvements and of course content
